### PR TITLE
Add an entry.

### PR DESCRIPTION
**Pull request by bangerth**

### CODE REVIEW

File: publications-2019.bib [hunk 1]
```diff
@@ -13,4 +13,20 @@ @Article{Basak2019a
...
... system using the discontinuous Galerkin method, following the work by Hu and Shu [SIAM J. Sci. Comput. 21 (1999) 660–690]; the missing constant is then recovered by the continuity of the level-set function while taking into account the characteristics. To handle highly distorted initial conditions, we develop a hybrid numerical flux that combines the Lax–Friedrichs flux and the penalty flux. Our method is stable for non-trivial test cases and handles singularities away from the interface very well. When derivative singularities are present on the interface, a second-derivative limiter is designed to suppress the oscillations. At least (N+1)th order accuracy in the interface cells and Nth order in the whole domain are observed for smooth solutions when Nth degree polynomials are used. Two dimensional test cases are presented to demonstrate superior properties such as accuracy, long-term stability, interface-preserving capability, and easy treatment of contact lines. We also show some preliminary results on the pinch-off process of a pendant drop, where topological changes of the fluid interface are involved. Our method is readily extendable to three dimensions and adaptive meshes."
```
Comment (jppelteret) [inline publications-2019.bib]: One of these hyphens is probably a dash. For simplicity, I suggest just removing the abstract entirely.

File: publications-2019.bib [hunk 2]
```diff
@@ -13,4 +13,20 @@ @Article{Basak2019a
   url       = {https://www.researchgate.net/profile/Anup_Basak/publication/326847352_Finite_element_procedure_and_simulations_for_a_multiphase_phase_field_approach_to_martensitic_phase_transformations_at_large_strains_and_with_interfacial_stresses/links/5baae99da6fdccd3cb733c04/Finite-element-procedure-and-simulations-for-a-multiphase-phase-field-approach-to-martensitic-phase-transformations-at-large-strains-and-with-interfacial-stresses.pdf},
 }
 
+@article{ZHANG2019634,
+  title = "A high-order and interface-preserving discontinuous Galerkin method for level-set reinitialization",
```
Comment (jppelteret) [inline publications-2019.bib]: For consistency, would you mind please changing the quotes used for all fields to braces? It helps protect against runaway arguments (something I had difficulty with when initially importing entries).

File: publications-2019.bib [hunk 3]
```diff
@@ -13,4 +13,20 @@ @Article{Basak2019a
...
+  journal = "Journal of Computational Physics",
+  volume = "378",
+  pages = "634 - 664",
```
Comment (jppelteret) [inline publications-2019.bib]: This _might_ be a dash instead of a hyphen.

File: publications-2019.bib [hunk 4]
```diff
@@ -13,4 +13,20 @@ @Article{Basak2019a
...
+  year = "2019",
+  issn = "0021-9991",
```
Comment (jppelteret) [inline publications-2019.bib]: This _might_ be a dash instead of a hyphen.

File: publications-2019.bib [hunk 5]
```diff
@@ -13,4 +13,20 @@ @Article{Basak2019a
...
+  doi = "10.1016/j.jcp.2018.11.029",
+  url = "http://www.sciencedirect.com/science/article/pii/S0021999118307733",
+  author = "Jiaqi Zhang and Pengtao Yue",
+  keywords = "Hamilton–Jacobi equation, Numerical flux, Second-derivative limiter, Weighted local projection, Moving contact line",
```
Comment (jppelteret) [inline publications-2019.bib]: > Hamilton–Jacobi; Second-derivative

This _might_ be a dash instead of a hyphen.

**Comment by bangerth** [comment]
Let's try again.

**Comment by tjhei** [comment]
> Let's try again.

nope. :-)

**Comment by bangerth** [comment]
Sigh... Maybe now?

**Comment by bangerth** [comment]
Sure, here you go!

**Pull request merged**